libsolidity/analysis/FunctionCallGraph.cpp

```diff
@@ -212,7 +212,7 @@ bool FunctionCallGraphBuilder::visit(ModifierInvocation const& _modifierInvocati
 bool FunctionCallGraphBuilder::visit(NewExpression const& _newExpression)
 {
 	if (ContractType const* contractType = dynamic_cast<ContractType const*>(_newExpression.typeName().annotation().type))
-		m_graph.createdContracts.emplace(&contractType->contractDefinition());
+		m_graph.createdContracts.emplace(&contractType->contractDefinition(), &_newExpression);
```

Comment:
Maybe rename this now from `createdContracts` to something like `bytecodeDependency`?